x-pack/plugins/serverless_search/public/application/components/connectors/connector_config/connector_config_panels.tsx

```diff
@@ -46,6 +48,16 @@ export const ConnectorConfigurationPanels: React.FC<ConnectorConfigurationPanels
       <EuiPanel hasBorder>
         <ConnectorIndexnamePanel connector={connector} />
       </EuiPanel>
+      <EuiSpacer />
+      <ConnectionDetails
+        connectorId={connector.id}
+        serviceType={connector.service_type}
+        status={connector.status}
+      />
+      <EuiSpacer />
+      <EuiSpacer />
```

Comment:
did you put 2 `<EuiSpacer />` here in purpose vs using a bigger size?

Reply:
No just a copy-paste error, fixed now :)